spacy/training/initialize.py

```diff
@@ -95,7 +95,7 @@ def load_vectors_into_model(
     """Load word vectors from an installed model or path into a model instance."""
     try:
         vectors_nlp = load_model(name)
-    except ConfigValidationError as e:
+    except ConfigValidationError:
```

Comment:
Oh, the real bug here was that `ConfigValidationError.from_error` should take `e` as the first argument